onnxruntime/core/framework/allocatormgr.cc

```diff
@@ -22,8 +22,13 @@ using namespace ::onnxruntime::common;
 AllocatorPtr CreateAllocator(DeviceAllocatorRegistrationInfo info, OrtDevice::DeviceId device_id) {
   auto device_allocator = std::unique_ptr<IDeviceAllocator>(info.factory(device_id));
   if (device_allocator->AllowsArena()) {
+#ifdef USE_MIMALLOC
     return std::shared_ptr<IArenaAllocator>(
-          onnxruntime::make_unique<TArenaAllocator>(std::move(device_allocator), info.max_mem));
+          onnxruntime::make_unique<MiMallocArena>(std::move(device_allocator), info.max_mem, arena_extend_strategy));
+#else
```

Comment:
should the last param be info.arena_extend_strategy?

Reply:
does the MiMallocArena class need updating for the addition param (or maybe it shouldn't be passed that param at all)?

---
In reply to: [396966796](https://github.com/microsoft/onnxruntime/pull/3174#discussion_r396966796) [](ancestors = 396966796)